Misc/NEWS.d/next/Library/2025-01-21-11-48-19.gh-issue-129027.w0vxzZ.rst

```diff
@@ -0,0 +1 @@
+Raise DeprecationWarning for :func:`sys._clear_type_cache`
```

Comment:
We could also have a What's New entry as well, not just a NEWS entry. You could also mention that this function was deprecated since 3.13 but didn't raise a deprecation warning at runtime.